index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/452c25fd08.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="css/style.css" />
+  <title>Landing Page</title>
+</head>
+<body>
+<header id="header">
+	<img id="header-img" src="https://s3.amazonaws.com/freecodecamp/original_trombones.png" alt="hello world">
+	<nav id="nav-bar"> 
+		 <ul>
+        <li>
+          <a class = "nav-link" href = "#features">Features</a>
+        <li>
+          <a class = "nav-link" href = "#how-it-works">How it Works</a>
+        <li>
+          <a class = "nav-link" href = "#pricing">Pricing</a>
+      </ul>
+	</nav>
+</header>
+<div class="form">
+  
+<form action="https://www.freecodecamp.com/email-submit">
+  <h2>Handcrafted, home-made masterpieces Get intouch </h2>
+    <input id="email" type="email" placeholder="Enter Your Email required">
+    <input id="submit" type="submit" value="Get Started">
+  </form>
+	</div>
+  <div class="main">
```

Comment:
Your form looks great, for the next part I think it would be better to use semantic tags instead of `div`. ✌️